### ISSUE TITLE
nguibs-modal - supports all modal sizes

### ISSUE THREAD

**Issue by lucienbertin** [LuccaSA/lucca-ui#120]
now the modal only supports size `lg` and i need a small one

see `\scss\plugins\angular-ui-bootstrap-reskin\components\_modal.scss` line ~25

```sass
// Sizing: lg, md & sm
// A large modal has the same size as a container
&.modal-lg {
	@each $device, $props in luiTheme(reference, breakpoints) {
		@media screen and (min-width: map-get($props, break)) {
			margin-left: map-get($props, spacing);
			margin-right: map-get($props, spacing);
		}
	}
}
``` 

but a `modal-sm` has a width of `100%` on a 1080x1920 screen

**Comment by bpaugam** [comment]
Fixed in the 2.0.8 release

**Comment by lucienbertin** [comment]
its not diplaying atrociously but a modal `sm` has the same width as a modal `lg`

**Comment by lucienbertin** [comment]
fixed in release [2.0.9](https://github.com/LuccaSA/lucca-ui/releases/tag/2.0.9) - see [demo](http://luccasa.github.io/lucca-ui/#/nguibs#modal) for more info
